docs/src/tutorials/getting_started/transitioning_from_matlab.jl

```diff
@@ -0,0 +1,389 @@
+
+# ## Vectorization
+
+# In MATLAB, it is absolutely essential to "vectorize" your code to obtain
+# acceptable performance. This is because MATLAB is a slow interpreted
+# language, which sends your commands to fast libraries. When you "vectorize"
+# your code you are minimizing the MATLAB part of the work and sending it to the
+# fast libraries instead.
+
+# There's no such duality with Julia.
+
+# Everything you write and most libraries you use will compile down to LLVM, so
+# "vectorization" has no effect.
+
+# For example, if you are writing a linear program in MATLAB and instead of the
+# usual `constraints = [v >= 0]` you write:
+# ```matlab
+# for i = 1:n
+#    constraints = [constraints, v(i) >= 0];
+# end
+# ```
+# performance will be poor.
+
+# With Julia, on the other hand, there is hardly any difference between
+# ```julia
+# @constraint(model, v >= 0)
+# ```
+#  and
+# ```julia
+# for i in 1:n
+#     @constraint(model, v[i] >= 0)
+# end
+# ```
+
+# ## Rosetta stone
+
+# To finish this tutorial, we show a complete example of the same optimization
+# problem being solved with JuMP, YALMIP, and CVX. It is a semidefinite program
+# that computes a lower bound on the random robustness of entanglement using the
+# partial transposition criterion.
+
+# The code is complete, apart from the function that does partial transposition.
+# With both YALMIP and CVX we use the function `PartialTranspose` from
+# [QETLAB](https://github.com/nathanieljohnston/QETLAB). With JuMP, we could use
+# the function `Convex.partialtranspose` from [Convex.jl](https://jump.dev/Convex.jl/stable/),
+# but we reproduce it here for simplicity:
+
+function partial_transpose(x::AbstractMatrix, sys::Int, dims::Vector)
+    @assert size(x, 1) == size(x, 2) == prod(dims)
+    @assert 1 <= sys <= length(dims)
+    n = length(dims)
+    s = n - sys + 1
+    p = collect(1:2n)
+    p[s], p[n+s] = n + s, s
+    r = reshape(x, (reverse(dims)..., reverse(dims)...))
+    return reshape(permutedims(r, p), size(x))
+end
+
+# ### JuMP
+
+# The JuMP code to solve this problem is:
+
+using JuMP
+import Clarabel
+import LinearAlgebra
+
+function random_state_pure(d)
+    x = randn(Complex{Float64}, d)
+    y = x * x'
+    return LinearAlgebra.Hermitian(y / LinearAlgebra.tr(y))
+end
+
+function robustness_jump(d)
+    rho = random_state_pure(d^2)
+    id = LinearAlgebra.I(d^2)
+    rhoT = partial_transpose(rho, 1, [d, d])
+    model = Model()
+    @variable(model, λ)
+    @constraint(
+        model,
+        PPT,
+        LinearAlgebra.Hermitian(rhoT + λ * id) in HermitianPSDCone(),
+    )
+    @objective(model, Min, λ)
+    set_optimizer(model, Clarabel.Optimizer)
+    set_attribute(model, "verbose", true)
+    optimize!(model)
+    if is_solved_and_feasible(model)
+        WT = dual(PPT)
+        return value(λ), LinearAlgebra.dot(WT, rhoT)
+    else
+        return "Something went wrong: $(raw_status(model))"
+    end
+end
```

Comment:
With  #3695 merged maybe you could go back to the previous version? I think it's much nicer.
```suggestion
function robustness_jump(d)
    rho = random_state_pure(d^2)
    id = LinearAlgebra.Hermitian(LinearAlgebra.I(d^2))
    rhoT = LinearAlgebra.Hermitian(partial_transpose(rho, 1, [d, d]))
    model = Model()
    @variable(model, λ)
    @constraint(model, PPT, rhoT + λ * id in HermitianPSDCone())
    @objective(model, Min, λ)
    set_optimizer(model, Clarabel.Optimizer)
    set_attribute(model, "verbose", true)
    optimize!(model)
    if is_solved_and_feasible(model)
        WT = dual(PPT)
        return value(λ), LinearAlgebra.dot(WT, rhoT)
    else
        return "Something went wrong: $(raw_status(model))"
    end
end
```

Reply:
Done